kolibri/plugins/learn/assets/src/views/LearnIndex.vue

```diff
@@ -16,17 +23,30 @@
   import NotificationsRoot from 'kolibri.coreVue.components.NotificationsRoot';
   import plugin_data from 'plugin_data';
   import { PageNames } from '../constants';
+  import PostSetupModalGroup from '../../../../device/assets/src/views/PostSetupModalGroup.vue';
```

Comment:
Ideally, I'd want to move this to `kolibri-common` rather than have so many `..` - but we can also move all this cleanup to a follow up if you'd prefer.